### PR TITLE
Update documentation about consume

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

**Review by tjtelan — APPROVED** [review]
LGTM!

File: content/cli/commands/consume.md [hunk 1]
```diff
@@ -120,7 +121,10 @@ Records that were not given a key are printed with `[null]`.
 Fluvio SmartModules are WASM modules that can edit the contents of a stream
 inline, before the records of that stream are delivered to a consumer. In order
 to use SmartModules, you must supply the WASM module to the `fluvio consume`
-command using the SmartModule options: `--filter`, `--map`, `--aggregate`.
+command using the SmartModule options: `--filter`, `--map`, `--aggregate`, `--filter-map`, `--array-map`.
+
+For Fluvio CLI >= 0.9.31 you can use the `--smartmodule` flag to specify the smartmodule name.
```
Comment (morenol) [inline content/cli/commands/consume.md:126]:
```suggestion
For Fluvio CLI >= 0.9.32 you can use the `--smartmodule` flag to specify the smartmodule name.
```

File: content/cli/commands/consume.md [hunk 2]
```diff
@@ -136,6 +140,8 @@ can apply it to the consumer as follows:
 %copy first-line%
 ```bash
 $ fluvio consume my-topic -B --filter="fluvio_wasm_filter.wasm"
+# or with the --smartmodule flag for fluvio 0.9.31 or later
```
Comment (morenol) [inline content/cli/commands/consume.md:143]:
```suggestion
# or with the --smartmodule flag for fluvio 0.9.32 or later
```

**Review by morenol — COMMENTED** [review]
released in 0.9.32

**Pull request merged**